libs/explorers/util/src/lib/loading-container/loading-container.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="loading-container">
+  <explorers-loading-icon
+    [colorInnermost]="colorInnermost"
+    [colorCentral]="colorCentral"
+    [colorOutermost]="colorOutermost"
+  ></explorers-loading-icon>
+  <h1>Loading {{ count }} results</h1>
```

Comment:
The design has slightly different language and uses `<h2>` for this element, so the font is DM Sans Regular instead of Bold -- 

text in loom | text in design 
|:---:|:---:|
| <img width="435" alt="loom" src="https://github.com/user-attachments/assets/22e03d51-c25e-4a2c-9ab5-b228ed5f76ee" /> | <img width="357" alt="figma" src="https://github.com/user-attachments/assets/097af8f3-da2b-40da-88e9-281e948a9a73" /> |

Let's try something like this? 

```suggestion
  <h2>Loading {{ count }} results for you</h2>
```

Reply:
good catch.  I think I'll change this to a <p> tag since header tags should ideally be for section headings and this is just a standalone piece of text.  I'll go with a font-weight of 300 to try and approximate the Figma design as the design has normal weight and that is not what we see.